### PR TITLE
clarifies README

### PR DESCRIPTION
**Pull request by DougReeder**
Clarifies a few things that aren't obvious to newcomers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Hubs Compose
 
 Hubs Compose is a Docker Compose setup than can be used to orchestrate all the
-services used by Hubs for local development.[^1]
+services used by Hubs for local development.[^1]  It runs on Windows, MacOS or Linux.
```
Comment (Exairnous) [inline README.md:4]: This would be good to include, I think.

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ certificates, you can visit https://hubs.local:4000 from your browser.
 ### Initial Setup
 
 1. [Install Docker Compose](https://docs.docker.com/compose/install)
-2. [Install Mutagen](https://mutagen.io/documentation/introduction/installation)
+2. [Install Mutagen](https://mutagen.io/documentation/introduction/installation) [The scripts start the Mutagen daemon, so it is *not* necessary to configure your system to automatically start on boot.]
```
Comment (Exairnous) [inline README.md:46]: This would be good to include, I think.

**Review by Exairnous — CHANGES_REQUESTED** [review]
I think most of this got covered with https://github.com/Hubs-Foundation/hubs-compose/pull/60 but the two lines I've marked with in-line comments would be good to add.  Could you rebase this on the latest main branch and condense it to just those two changes, please?

File: README.md (hunk 1 above)
Comment (DougReeder) [reply README.md]: done

**Review by Exairnous — APPROVED** [review]
Looks great.  Thanks!  Nice idea adding links to jump to the referenced sections.  Merging.

**Pull request merged**